src/utils/computeFiatValuePriceImpact.ts

```diff
@@ -0,0 +1,21 @@
+import { CurrencyAmount, Percent, _10000, JSBI, currencyEquals, ZERO } from '@swapr/sdk'
+
+const ONE_HUNDRED_PERCENT = new Percent(_10000, _10000)
+
+/**
+ * Calculates the difference between fiat value ouput and input in percentage terms.
+ * @param {CurrencyAmount} fiatValueInput
+ * @param {CurrencyAmount} fiatValueOutput
+ * @returns {Percent}
+ */
+export function computeFiatValuePriceImpact(
+  fiatValueInput: CurrencyAmount | undefined | null,
+  fiatValueOutput: CurrencyAmount | undefined | null
+): Percent | undefined {
+  if (!fiatValueOutput || !fiatValueInput) return undefined
+  if (!currencyEquals(fiatValueInput.currency, fiatValueOutput.currency)) return undefined
+  if (JSBI.equal(fiatValueInput.quotient, ZERO)) return undefined
```

Comment:
Could this be consolidated into one statement with OR operators? like this 
`if ((!fiatValueOutput || !fiatValueInput) || !currencyEquals(fiatValueInput.currency, fiatValueOutput.currency) || JSBI.equal(fiatValueInput.quotient, ZERO))`

Reply:
Fixed!